src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -107,6 +107,16 @@ Useful for multimodal co-registration with MEG, (S)EEG, TMS, and so on.
 | ----------------------------- | --------------------- | ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | AnatomicalLandmarkCoordinates | RECOMMENDED           | [object][] of [arrays][] | Key:value pairs of any number of additional anatomical landmarks and their coordinates in voxel units (where first voxel has index 0,0,0) relative to the associated anatomical MRI (for example, `{"AC": [127,119,149], "PC": [128,93,141], "IH": [131,114,206]}`, or `{"NAS": [127,213,139], "LPA": [52,113,96], "RPA": [202,113,91]}`). Each array MUST contain three numeric values corresponding to x, y, and z axis of the coordinate system in that exact order. |
 
+### Curation details
```

Comment:
Should this go under the Anatomical MRI section or is it generic across MRI data types?

Reply:
Skull-stripping can in principle be done to any MR image, since there is signal outside the brain that can show up in many sequences (e.g., the eyes show up quite clearly in EPI). Defacing is not well defined in non-anatomical modalities, but it seemed appropriate to keep these together.

What if I changed to `RECOMMENDED for anatomical images`?

Reply:
If it's feasible for all MR image types, then I'm good with it as-is. Thanks for the clarification.